src/controllers/best-practice.js

```diff
@@ -7,7 +7,14 @@ const bestPracticeController = (request) => {
     request.session.bestPracticeError = false;
     // Save the agreement to comply.
     request.session.bestPractice = true;
-    // Follow the 'happy path'.
+    // The request passed all our validation, we've stored copies of everything we
+    // need, so it's time to go on.
+    if (request.session.qualificationHeld === 'dsc1') {
```

Comment:
The comment and the code don't match. The comment indicates the positive path, but the code's taking us down the secondary one.

Reply:
Resolved in 835c3f6e21d690211769853ecc625e19d00292c6.